doc/source/conf.py

```diff
@@ -179,6 +187,12 @@ def setup(app):
   "external_links": [
       {"name": "Learn", "url": "https://numpy.org/numpy-tutorials/"}
       ],
+  # Add light/dark mode and documentation version switcher:
+  "navbar_end": ["version-switcher", "navbar-icon-links"],
```

Comment:
Adding the dark-mode should just mean inserting it here.  But I don't think the dark-mode is released yet.